frontend/src/features/backoffice/control_objectives/ControlObjectives.tsx

```diff
@@ -45,7 +45,8 @@ export function ControlObjectives() {
   )
 
   useEffect(() => {
-    dispatch(getAllFleetSegments())
```

Comment:
Do not forget to replace all calls to `fleetSegment` redux state

Reply:
There is no need for that anymore since they all use RTK cache instead of "caching" them in the store.